dear-imgui.cabal

```
@@ -97,3 +129,34 @@ executable readme
   default-language: Haskell2010
   build-depends: base, sdl2, gl, dear-imgui, managed
   ghc-options: -Wall
+
+executable vulkan
+  main-is: Main.hs
+  other-modules: Attachments, Backend, Input, Util
+  hs-source-dirs: examples/vulkan
+  default-language: Haskell2010
+  build-depends:
+      dear-imgui
+    , base
+        >= 4.13 && < 4.16
+    , bytestring
+        >= 0.10.12.0 && < 0.12
```

I had to relax this locally to get this to build:

```suggestion
        >= 0.10.10.0 && < 0.12
```